presto-main/pom.xml

```diff
@@ -407,6 +407,9 @@
                     <excludes>
                         <exclude>**/TestLocalExecutionPlanner.java</exclude>
                     </excludes>
+                    <systemPropertyVariables>
+                        <org.joda.time.DateTimeZone.Provider>com.facebook.presto.tz.JdkBasedZoneInfoProvider</org.joda.time.DateTimeZone.Provider>
```

Comment:
Does joda support service loader? Or any other way to make this be loaded more reliably?

Reply:
https://github.com/JodaOrg/joda-time/blob/master/src/main/java/org/joda/time/DateTimeZone.java#L507-L548